server/routerlicious/packages/services-telemetry/src/resources.ts

```diff
@@ -0,0 +1,37 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Lumber } from "./lumber";
+
+export enum LogLevel {
+    Error,
+    Warning,
+    Info,
+    Verbose,
+    Debug,
+}
+
+export enum LumberType {
+    Metric,
+    Log,
+}
+
+export interface ILumberjackEngine {
+    emit(lumber: Lumber<string>): void;
+}
+
```

Comment:
Silly question, but is the existing [ITelemetryLogger](https://github.com/microsoft/FluidFramework/blob/main/common/lib/common-definitions/src/logger.ts) on your radar?

(@VladSud - FYI)

Reply:
I had conversations with @tanviraumi and @markfields and we decided to not invest into integrating with `ITelemetryLogger` at this point. Client and service logging are still quite different, and given that we plan to use OpenTelemetry once it becomes mature, that could be a more appropriate opportunity to both leverage OpenTelemetry and try to unify logging on service and client. Having said that, I will keep an eye on the TelemetryLogger classes to make sure we can have a consistent E2E story - for example, being able to correlate client and service side logs.